manual.py

```diff
@@ -0,0 +1,568 @@
+from cdp_backend.pipeline import ingestion_models
+from datetime import datetime
+
+event = ingestion_models.EventIngestionModel(
+    body=ingestion_models.Body("CITY OF HOUSTON . CITY COUNCIL", is_active=True),
+    sessions=[
+        ingestion_models.Session(
+            # session 0 no video, combine with session 1?????????????
+            video_uri="https://houstontx.new.swagit.com/videos/176697/embed",
+            session_index=1,
+            session_datetime='June 15, 2022'
+        )
+    ],
+    event_minutes_items = [
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PRESENTATIONS")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("ROLL CALL AND ADOPT THE MINUTES OF THE PREVIOUS MEETING")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("MAYOR'S REPORT")
+        ),
+        # where should i put ACCEPT WORK???? following two belongs to accept work but not matters
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,949,927.38 and acceptance of work on contract with REYTEC CONSTRUCTION RESOURCES, INC for FY2018 Drainage Rehab Work Orders #4 - 1.67% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS and K - CASTEX-TATUM")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,979,476.73 and acceptance of work on contract with GRAVA LLC for FY2019 Open Drainage System Maintenance Work Orders #1 - 0.68% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS; J - POLLARD and K - CASTEX-TATUM")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PROPERTY - NUMBER 3")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PURCHASING and TABULATION OF BIDS - NUMBERS 4 through 9")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("ORDINANCES - NUMBERS 10 through 26"),
```

Comment:
As mentioned yesterday, these are simply just headings not minutes items or matters. Each of the sub items are the minutes items and matters discussed.